var/spack/repos/builtin/packages/fastqc/package.py

```diff
@@ -41,15 +39,14 @@ class Fastqc(Package):
     patch('fastqc.patch', level=0)
 
     def install(self, spec, prefix):
-        mkpath(self.prefix.bin)
-        mkpath(self.prefix.lib)
-        copy_file('fastqc', self.prefix.bin)
+        mkdirp(self.prefix.bin)
+        mkdirp(self.prefix.lib)
+        install('fastqc', self.prefix.bin)
+        set_executable(join_path(self.prefix.bin, 'fastqc'))
         for j in ['cisd-jhdf5.jar', 'jbzip2-0.9.jar', 'sam-1.103.jar']:
-            copy_file(j, self.prefix.lib)
+            install(j, join_path(self.prefix.lib, j))
         for d in ['Configuration', 'net', 'org', 'Templates', 'uk']:
-            copy_tree(d, join_path(self.prefix.lib, d))
-        chmod = which('chmod')
-        chmod('+x', join_path(self.prefix.bin, 'fastqc'))
+             install_tree(d, join_path(self.prefix.lib, d))
```

Comment:
This line is indented one space too far and is causing the flake8 tests to fail.